### ISSUE TITLE
RLInterface.jl and POMDPReinforce.jl

### ISSUE THREAD

**Issue by rcnlee** [JuliaPOMDP/RLInterface.jl#17]
What is the difference between these two RL interfaces?  Should this one be used for new projects?

**Comment by zsunberg** [comment]
Yeah, I think REINFORCE.jl has been abandoned. This one is the one to use
